gubernator/log_parser.py

```diff
@@ -64,7 +64,14 @@ def digest(data, skip_fmt=lambda l: '... skipping %d lines ...' % l,
             previous_end = 0
         skip_amount = match - previous_end - CONTEXT
         if skip_amount > 1:
-            output.append('<span class="skip">%s</span>' % skip_fmt(skip_amount))
+            output.append('<span class="skip"><a href="javascript:show_skipped(\'skip_%s\')"'
```

Comment:
nit: declare a local variable with the full id in it. E.g. `id = 'skip_%s' % match`, and use that here and below. This just makes it a bit more robust to errors.